gtsfm/common/image.py

```diff
@@ -48,27 +54,60 @@ def get_intrinsics_from_exif(self) -> Optional[Cal3Bundler]:
             intrinsics matrix (3x3).
         """
 
-        if self.exif_data is None or len(self.exif_data) == 0:
-            return None
-
-        focal_length_mm = self.exif_data.get("FocalLength")
-
-        if focal_length_mm is None:
-            return None
-
-        try:
-            sensor_width_mm = Image.sensor_width_db.lookup(
-                self.exif_data.get("Make"),
-                self.exif_data.get("Model"),
-            )
-        except (AttributeError, LookupError):
-            # TODO(yanwei.du): Set a prior focal length, e.g. 1.5 * max(self.width, self.height).
-            return None
-
         img_w_px = self.width
         img_h_px = self.height
-        focal_length_px = max(img_h_px, img_w_px) * focal_length_mm / sensor_width_mm
-
+        max_size = max(img_w_px, img_h_px)
+
+        # Initialize focal length by `default_focal_length_factor * max(width, height)`.
+        focal_length_px = self.default_focal_length_factor * max_size
+
+        # Read focal length prior from exif.
+        if self.exif_data is not None and len(self.exif_data) > 0:
+
+            # Read from `FocalLengthIn35mmFilm`.
```

Comment:
If it's not too much work, could we add some small tests for the new code paths through this function?

Reply:
Done

Reply:
Marking as resolved, thanks!